tools/test/config/feature_recursive_add/FEATURE_STORAGE/lib2/mbed_lib.json

```diff
@@ -1,8 +1,3 @@
 {
-    "name": "lib2",
```

Comment:
Removing ` tools/test/config/feature_recursive_add` test as it uses uvisor feature, wont need any replacement?